android/app/src/main/kotlin/com/jabook/app/jabook/audio/AudioSessionSetup.kt

```
@@ -0,0 +1,126 @@
+                    putBoolean(androidx.media3.session.MediaConstants.EXTRAS_KEY_SLOT_RESERVATION_SEEK_TO_NEXT, true)
+                }
+            service.mediaSession = service.mediaLibrarySession
+
+            android.util.Log.i(
+                "AudioPlayerService",
+                "MediaLibrarySession created successfully: ${service.mediaLibrarySession?.token}",
+            )
+
+            createServiceMediaController()
+            service.mediaSessionManager = MediaSessionManager(service, service.exoPlayer)
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is too generic. Prefer catching specific exceptions to the case that is currently handled.

(detekt.exceptions.TooGenericExceptionCaught)